tests/no-viscosity-in-adiabatic-conditions.cc

```diff
@@ -1,80 +0,0 @@
-#include <aspect/material_model/simple.h>
```

Comment:
The test assumed that it was possible to query the material model's viscosity independent of the other state variables. This is no longer possible with the new interface, so the test doesn't really make sense anymore. Misuse of uninitialized variables should be caught by your FPE thing now anyway.